fintech-examples/fintech-impl/src/main/java/de/adorsys/opba/fintech/impl/service/AccountService.java

```diff
@@ -33,6 +33,10 @@
 @Slf4j
 @RequiredArgsConstructor
 public class AccountService {
+    //TODO Fix hardcoded values
+    private static final int PAGE = 1;
+    private static final int TRANSACTIONS_PER_PAGE = 500;
```

Comment:
Passing pageSize as null should force returning all (or provider default value - i.e. 500 for the certain provider) transactions. So this should be null and unnecessary

Reply:
(duplicated comment)
Deleted unnecessary vars, changed fields to null.